app/editor/src/components/form/wysiwyg/Wysiwyg.tsx

```diff
@@ -44,7 +52,7 @@ export const Wysiwyg: React.FC<IWysiwygProps> = ({ fieldName, label, required, e
 
   React.useEffect(() => {
     if (!!id) {
-      setState({ ...state, html: values[fieldName] as string });
+      setState({ ...state, html: values[fieldName!] as string });
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You should at least at `fieldName` to the dependencies.  This highlights one of the issues when we ignore the warning.

Reply:
Updated, thanks